docker/ebpf-probe-builder/build_bpf_probe.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -euo pipefail
+
+# Defaults
+DRIVER_DIR=/opt/draios/src/draios-agent-0.1.1dev
+KERNEL_DIR=/lib/modules/$(uname -r)/build
+OUT_DIR=${HOME}/.sysdig
+
+usage()
+{
+	echo "build_bpf_probe [-d <driver_path>] [-k <kernel_path>] [-o <output_path>]"
+}
+
+# Options parsing
+while [ -n "${1-}" ]; do
+	case $1 in
```

Comment:
Minor: this file seems to be using both tabs and spaces for indentation.

Reply:
I used tabs for indentation and spaces for formatting. Anything indented should have a leading tab.